src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingAssessmentIntegrationTest.java

```diff
@@ -91,25 +91,21 @@ void initTestCase() {
 
         programmingExerciseStudentParticipation = database.addStudentParticipationForProgrammingExercise(programmingExercise, "student2");
         // A new manual result and submission are created during the locking of submission for manual assessment
-        // The new result has a result string, assessment type, automatic feedbacks and assessor
+        // The new result has an assessment type, automatic feedbacks and assessor
         var automaticFeedback = new Feedback().credits(0.0).detailText("asdfasdf").type(FeedbackType.AUTOMATIC).text("asdf");
         var automaticFeedbacks = new ArrayList<Feedback>();
         automaticFeedbacks.add(automaticFeedback);
-        var newManualResult = database.addResultToParticipation(AssessmentType.SEMI_AUTOMATIC, null, programmingExerciseStudentParticipation, "2 of 3 passed", "tutor1",
-                automaticFeedbacks);
+        var newManualResult = database.addResultToParticipation(AssessmentType.SEMI_AUTOMATIC, null, programmingExerciseStudentParticipation, "tutor1", automaticFeedbacks);
         programmingExerciseStudentParticipation.addResult(newManualResult);
         // Set submission of newResult
         database.addProgrammingSubmissionToResultAndParticipation(newManualResult, programmingExerciseStudentParticipation, "123");
 
-        manualResult = ModelFactory.generateResult(true, 90D).participation(programmingExerciseStudentParticipation);
         List<Feedback> feedbacks = ModelFactory.generateFeedback().stream().peek(feedback -> feedback.setDetailText("Good work here")).collect(Collectors.toList());
```

Comment:
```suggestion
        List<Feedback> feedbacks = ModelFactory.generateFeedback().stream().peek(feedback -> feedback.setDetailText("Good work here")).toList();
```

Reply:
This is currently used in many more places and these two methods are in fact not the same, since .toList() returns an unmodifiable List while Collectors.toList() returns an ArrayList. Therefore I'd suggest changing this in a separate PR and verifying it there that the behavior stayed the same.